src/rust/ide/src/model/execution_context.rs

```diff
@@ -17,6 +20,11 @@ use enso_protocol::language_server;
 #[fail(display="Tried to pop an entry point")]
 pub struct PopOnEmptyStack {}
 
+/// Error when using an Id that does not correspond to any known visualization.
+#[derive(Clone,Copy,Debug,Fail)]
+#[fail(display="Tried to use incorrect visualization Id")]
```

Comment:
Errors should be sentences for users. Thus, they need dot on the end.

Reply:
As far a I remember, after this message comes `:`, and more elaborate error message. So it's displayed as valid sentence.